doc/nrf/known_issues.rst

```diff
@@ -210,6 +210,14 @@ Subsystems
 |BLE|
 =====
 
+.. rst-class:: v1-4-0
```

Comment:
Unsure about how these versions are handled, the problem exists for all releases I would think.